exercises/1-the-manual-menace/README.md

```diff
@@ -304,21 +344,25 @@ ansible-playbook apply.yml -e target=tools \
 ### Part 4 - Commit CI/CD
 
 1. Navigate to GitLab login page. You can login using your cluster credentials using the LDAP tab
-![gitlab-ui](../images/exercise1/gitlab-ui.png)
+   ![gitlab-ui](../images/exercise1/gitlab-ui.png)
 
 2. Once logged in create a new project called `enablement-ci-cd` and mark it as internal. Once created, copy out the `git url` for use on the next step.
-![gitlab-new-project](../images/exercise1/gitlab-new-project.png)
+   ![gitlab-new-project](../images/exercise1/gitlab-new-project.png)
 
 3. Commit your local project to this new remote by first removing the existing origin (github) where the Ansible project was cloned from in the first steps. Remember to substitute `<GIT_URL>` accordingly with the one created for your `enablement-ci-cd` repository a moment ago.
```

Comment:
Could be good to add the following: You can find the `<GIT_URL>` by clicking on `Clone` and then copying the link under `Clone with HTTPS`.